node-fs-readfile/cat.js

```diff
@@ -0,0 +1,7 @@
+const fs = require('fs');
+for (let i = 2; i < process.argv.length; i++) {
+  fs.readFile(`./${process.argv[i]}`, 'utf8', (err, data) => {
+    if (err) throw err;
+    console.log(data);
+  });
+}
```

Comment:
Good attempt! This will print out all the files the user inputs, however there is no guarantee the output will be in the correct order.